indy_node/test/helper.py

```diff
@@ -186,7 +186,7 @@ def sdk_send_and_check_auth_rule_request(
 
     # temp fix untill new sdk released
     req_json = json.loads(req_json)
-    if req_json[OPERATION][CONSTRAINT][CONSTRAINT_ID] == 'ROLE':
+    if req_json[OPERATION][CONSTRAINT][CONSTRAINT_ID] == 'ROLE' and OFF_LEDGER_SIGNATURE in constraint:
```

Comment:
Since SDK supports OFF_LEDGER_SIGNATURE  now, do we still need this temp fix?

Reply:
Changes in https://github.com/hyperledger/indy-node/pull/1418